### PR TITLE
snow

### PR DESCRIPTION
**Pull request by valeIT**

### CODE REVIEW

**Review by MsSlacker — DISMISSED** [review]
Bit short and no conclusion ;P

File: _posts/2019-12-28-snow.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+It was reassuring to see that everyone had cleaned their cars from the snow before driving, that does not happen in Italy since people are not used to snow so as soon as they enter the highway you have snow and hide dangerously falling off cars and trucks(!) at 100+ km/h.
+
+I expect it not to be as fun if I had to use the car every day to go to work and I had to repeat this process for weeks at a time, but fortunately I walk to the office.
+
+
+## Swiss Highways
+The most snow we've seen was in the Italian/Swiss alps while coming back from Italy this november by car. There was so much snow that a lot of roads were closed and you could not go faster than 30 km/h on the highway in some areas. A lot of cars got stuck going uphill even though there were lots of snowplows in action.
+
+## Previous Years
+Last time I went to poland in winter there were 15 cm of snow, more than I've seen in Italy (apart from in the mountain region) in years. It has been hard to even move the car for a few km to go shopping. That year there were small piles of snow that turned into ice at the sides of the roads until march/april.
```
Comment (valeIT) [inline _posts/2019-12-28-snow.md]: Should the bottom be split in a different article? They seem to disjointed as they are now

**Comment by valeIT** [comment]
> Bit short and no conclusion ;P

Go to chnged files and see comment

File: _posts/2019-12-28-snow.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+categories: [General]
+image:
+image2:
+author: Valentino Urbano
+---
+
+After coming to Poland this summer I expected to see some snow by late Fall. Apart from the really cold temperatures, especially if you compared to what we're used to have in Italy, the weather was nothing special: sun, rain and clouds.
+
+No snow at all.
+Finally today we woke up with a really small layer of snow.
```
Comment (MsSlacker) [inline _posts/2019-12-28-snow.md]: Today - > the other day (since no longer applicable lol)

File: _posts/2019-12-28-snow.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+status: publish
+categories: [General]
+image:
+image2:
+author: Valentino Urbano
+---
+
+After coming to Poland this summer I expected to see some snow by late Fall. Apart from the really cold temperatures, especially if you compared to what we're used to have in Italy, the weather was nothing special: sun, rain and clouds.
+
+No snow at all.
```
Comment (MsSlacker) [inline _posts/2019-12-28-snow.md:16]: Paragraph?

File: _posts/2019-12-28-snow.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+Finally today we woke up with a really small layer of snow.
+
+Most of the roads are clean so it is not the kind of snow that you forces you to stay home. Unfortunately it is not even the kind of snow that you can have some fun throwing it to your friends/family since it's too icy since the temperatures are too low, but it is still nice to see.
+
+It took me 15 minutes more to clean the car, but it was a fun experience. Driving on the icy road for 10 minutes (yes, it took more time to de-ice the car than the driving time) was not the most pleasant experience, but nowhere near the worst.
```
Comment (MsSlacker) [inline _posts/2019-12-28-snow.md]: US
       I 
          HELPEDDDDD

Comment (MsSlacker) [reply _posts/2019-12-28-snow.md]: Paragraph

File: _posts/2019-12-28-snow.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+It was reassuring to see that everyone had cleaned their cars from the snow before driving, that does not happen in Italy since people are not used to snow so as soon as they enter the highway you have snow and hide dangerously falling off cars and trucks(!) at 100+ km/h.
```
Comment (MsSlacker) [inline _posts/2019-12-28-snow.md]: hide - > ice

Comment (MsSlacker) [reply _posts/2019-12-28-snow.md]: Can say we almost got hit by that shit twice...

File: _posts/2019-12-28-snow.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+I expect it not to be as fun if I had to use the car every day to go to work and I had to repeat this process for weeks at a time, but fortunately I walk to the office.
```
Comment (MsSlacker) [inline _posts/2019-12-28-snow.md:26]: had - > have?

File: _posts/2019-12-28-snow.md (hunk 1 above)
Comment (MsSlacker) [reply _posts/2019-12-28-snow.md]: It really? Can say you miss it :P and then go on about swisssssd

Can link the vid I posted on YouTube of the highway.... Just a few minutes but it looks nice

Comment (MsSlacker) [inline _posts/2019-12-28-snow.md]: Poland! Have some shame!

**Review by MsSlacker — CHANGES_REQUESTED** [review]

File: _posts/2019-12-28-snow.md (hunk 1 above)
Comment (MsSlacker) [reply _posts/2019-12-28-snow.md]: https://www.youtube.com/watch?v=gREHBNwQvN8&feature=share&fbclid=IwAR37xPdkS_NLROeUWcojVHGQ8iJGFCXJGrR4D3xtR0THXRgIZUWp_5RCkvY

**Review by MsSlacker — APPROVED** [review]

**Pull request merged**